test/sample_ipam/admin.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import swapper
+
+from django.contrib import admin
+
+from django_ipam.admin import (AbstractIPAddressAdmin, AbstractSubnetAdmin)
+
+
+IPAddress = swapper.load_model("django_ipam", "IPAddress")
+Subnet = swapper.load_model("django_ipam", "Subnet")
+
+
+# @admin.register(IPAddress)
```

Comment:
try to make this work before merging

Reply:
Do we really need to register them in the sample app, because it's working without doing it.

Reply:
you can start without the sample app and then add it later, if you want to go this route, remove the directory entirely. Keep in mind this rule: commit only working code

Reply:
No actually it's working without registering them in the admin.py, that's why I commented it out, and I keeping the sample app to generate the migrations.

Reply:
of course, that's the main app. The sample app should be a modified version of django-ipam, so the model names should be different You could call them `SampleSubnet` and `SampleIpAddress` for example.

Reply:
Aah thanks, I understood now. :+1: